.github/labeler.yml

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+
+provider:GCP:
+  - airflow/**/gcp/*
```

Comment:
GCP also have code in `airflow/gcp` directory.  Will this work?

Reply:
No, it would not. We will have to add that. Also, we hit a roadblock, check this: https://lists.apache.org/thread.html/03a94f681c96195f0359ed795a044591e864c8a9367cfa883fab92ba%40%3Cdev.airflow.apache.org%3E

Reply:
Waiting on INFRA team now